src/main/java/com/linecorp/armeria/client/metrics/MetricCollectingClient.java

```diff
@@ -0,0 +1,39 @@
+package com.linecorp.armeria.client.metrics;
+
+import java.util.function.Function;
+
+import com.codahale.metrics.MetricRegistry;
+
+import com.linecorp.armeria.client.Client;
+import com.linecorp.armeria.common.metrics.DropwizardMetricConsumer;
+
+
+public final class MetricCollectingClient {
```

Comment:
https://github.com/line/armeria/pull/114#discussion_r53743519

I realized I left inconsistent comments. Perhaps we need to decide what the best practice is. /cc @synk